pkg/trace/config/apply.go

```diff
@@ -348,7 +348,17 @@ func (c *AgentConfig) loadDeprecatedValues() error {
 		c.BucketInterval = d * time.Second
 	}
 	if cfg.IsSet("apm_config.receiver_timeout") {
-		c.ReceiverTimeout = cfg.GetInt("apm_config.receiver_timeout")
+		d := time.Duration(cfg.GetInt("apm_config.receiver_timeout"))
+		c.ReceiverReadTimeout = d * time.Second
+		c.ReceiverWriteTimeout = d * time.Second
+	}
+	if cfg.IsSet("apm_config.receiver_read_timeout") {
+		d := time.Duration(cfg.GetInt("apm_config.receiver_read_timeout"))
+		c.ReceiverReadTimeout = d * time.Second
```

Comment:
why not in one line ?

```suggestion
		c.ReceiverReadTimeout  = time.Duration(cfg.GetInt("apm_config.receiver_read_timeout")) * time.Second
```